src/main/java/com/denizenscript/clientizen/events/ClientizenScriptEventRegistry.java

```diff
@@ -1,8 +1,10 @@
 package com.denizenscript.clientizen.events;
 
+import com.denizenscript.denizencore.events.ScriptEvent;
+
 public class ClientizenScriptEventRegistry {
 
 	public static void registerEvents() {
-
+		ScriptEvent.registerScriptEvent(KeyPressReleaseEvent.class);
```

Comment:
Looks like you forgot to register `ScreenOpenCloseEvent`? or is there a reason it's not being registered?